core/src/main/kotlin/com/playkids/onboarding/core/util/ChooseValue.kt

```diff
@@ -0,0 +1,7 @@
+package com.playkids.onboarding.core.util
+
+import com.movile.kotlin.commons.dynamodb.toAttributeValue
+
+class ChooseValue(private val intValue: Int?, private val floatValue: Float?) {
+    fun chooseToAttributeValue() = intValue?.toAttributeValue() ?: floatValue.toAttributeValue()
```

Comment:
You shouldn't be able to do this in the core module, so I think that you might be importing the dynamodb dependency somewhere.

About the toAttributeValue: it should be inside the DynamoExtensions file, I think that you implemented it here instead of an extensions there just because you weren't importing the core module, as you do it now, you might extract this function there (otherwise you will not be able to remove the dynamo depency from the core)

Finally, you might not need this class at all, I think that you just created it because you needed a way to send either a Int or a Float to the same function, and there are a couple of ways we can do it without this class:
1. Converting the float to Int. Usually, we store the price in cents, so we just multiply the Float by 100 and convert it to Int
2. Changing the function header to receive a Number. All numeric classes extend the Number superclass, so you could just use it instead of specifying the class. (If you take a look at the toAttributeValue extension that you are using here, it is an extension over Number)